tests/test_viz.py

```diff
@@ -17,10 +17,12 @@
 def attributes_explorer(helpers):
     # Get JSONLD file path
     path_to_jsonld = helpers.get_data_path("example.model.jsonld")
+    path_to_graph = helpers.get_data_path("example.model.pickle")
```

Comment:
If `AttributedExplorer` can accept inputs of the jsonld, the pickle, and both, then the tests should cover intitializing the objecct with those combinations. Explore parameterization for swapping the objects in place of each other for the tests/fixtures